app/src/main/java/kr/co/yogiyo/rookiephotoapp/edit/EditResultActivity.kt

```diff
@@ -0,0 +1,131 @@
+package kr.co.yogiyo.rookiephotoapp.edit
+
+import android.Manifest
+import android.content.Context
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.net.Uri
+import android.os.Build
+import android.os.Bundle
+import android.os.Environment
+import android.support.v4.app.ActivityCompat
+import android.support.v7.widget.Toolbar
+import android.util.Log
+import android.view.Menu
+import android.view.MenuItem
+import android.view.View
+import kotlinx.android.synthetic.main.activity_edit_result.*
+import kr.co.yogiyo.rookiephotoapp.BaseActivity
+import kr.co.yogiyo.rookiephotoapp.R
+import java.io.File
+import java.io.FileInputStream
+import java.io.FileOutputStream
+import java.util.*
+
+class EditResultActivity : BaseActivity() {
+
+    private var getEditPhotoUri: Uri? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_edit_result)
+
+        setView()
+        setSettingAndResultActionBar()
+    }
+
+    // UCropView setting
+    private fun setView() {
+        getEditPhotoUri = intent.data
+        getEditPhotoUri?.let {
+            iv_edited_photo.setImageURI(it)
+        } ?: showToast(R.string.dont_load_captured_photo)
+    }
+
+    private fun setSettingAndResultActionBar() {
+        setSupportActionBar(findViewById<View>(R.id.toolbar) as Toolbar)
```

Comment:
findViewById가 없이 쓸 수 있어요.

```suggestion
        setSupportActionBar(toolbar)
```

Reply:
변경했습니다